### PR TITLE
Uno: Fix crash

### PR DESCRIPTION
**Pull request by Distrib-ps**

### CODE REVIEW

File: server/chat-plugins/uno.ts [hunk 1]
```diff
@@ -246,8 +246,11 @@ export class UNO extends Rooms.RoomGame {
 				this.sendToRoom(`|raw|${Utils.escapeHTML(name)} has not picked a color, the color will stay as <span style="color: ${textColors[this.topCard.changedColor]}">${this.topCard.changedColor}</span>.`);
 			}
 		}
-		if (this.awaitUno === userid) this.awaitUno = null;
-
+		if (this.awaitUno === userid) this.awaitUno = null; {
```
Comment (mia-pi-git) [inline server/chat-plugins/uno.ts:249]: uh, what? Does this work locally?

Comment (Distrib-ps) [reply server/chat-plugins/uno.ts:249]: For me yes:
Now: http://prntscr.com/yvvqsd
Before: http://prntscr.com/yvvu2v

Comment (mia-pi-git) [reply server/chat-plugins/uno.ts:249]: Okay then.
```suggestion
		if (this.awaitUno === userid) this.awaitUno = null;
```

File: server/chat-plugins/uno.ts [hunk 2]
```diff
@@ -246,8 +246,11 @@ export class UNO extends Rooms.RoomGame {
...
+			if (!this.topCard) {
+				throw new Chat.ErrorMessage(`Unable to disqualify ${name}.`);
+			}
+		}
```
Comment (mia-pi-git) [inline server/chat-plugins/uno.ts:253]:
```suggestion
		if (!this.topCard) {
			throw new Chat.ErrorMessage(`Unable to disqualify ${name}.`);
		}
```

**Comment by Distrib-ps** [comment]
It's good like that? @mia-pi-git

**Comment by Zarel** [comment]
Who even worked on Uno recently? This seems like a pretty esoteric error message, too – why are we sometimes unable to disqualify players?

**Comment by Distrib-ps** [comment]
I’ve been working on it recently (including a pending pr https://github.com/smogon/pokemon-showdown/pull/8001).
And yes, maybe an explanation would be better. Maybe "The uno hasn't started yet." ?

**Pull request merged**